user_video.py

```diff
@@ -8,16 +8,37 @@
 
 #This handles per user video processing
 class UserVideo:
-    def __init__(self, lang="jp", disable_dialog=False, disable_translation=False, enable_cache=False, translate="", textDetector=None, debug_bbox=False):
+    def __init__(self, lang="jp", disable_dialog=False, disable_translation=False, enable_cache=False, translate="", textDetector=None, debug_bbox=False, eval=False):
         self.last_inboard_frame = None
         self.last_frame_count = 0
         self.crop_height = 71
 
-        self.frameProcessor = FrameProcessor(lang, disable_dialog,)
+        if eval is False:
+            self.frameProcessor = FrameProcessor(lang, disable_dialog)
 
-        self.video_stream = VideoStreamWithAnnotations(background_task=self.process_video_thread, background_task_args={"translate" : translate, 'enable_cache' : enable_cache},
-                                                    show_fps=True, crop_y_coordinate=None, frameProcessor=self.frameProcessor, textDetector=textDetector, debug_bbox=debug_bbox) #TODO crop should be set later by user
-        #video_stream.stop()
+            self.video_stream = VideoStreamWithAnnotations(
+                background_task=self.process_video_thread,
+                background_task_args={"translate": translate, 'enable_cache': enable_cache},
+                show_fps=True,
+                crop_y_coordinate=None,
+                frameProcessor=self.frameProcessor,
+                textDetector=textDetector,
+                debug_bbox=debug_bbox
+            )
+
+        if eval is True:
+            self.frameProcessor_jp = FrameProcessor(language='jp', disable_dialog=disable_dialog)
```

Comment:
the evaluation code shouldn't leak into the production code. Can you just create 2 instances of user_video one with EN and one with JP?

Reply:
I've reverted the user_video.py code to previous version. removed all the eval related code.